feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/TimetableItemDetailViewModel.kt

```diff
@@ -52,18 +53,28 @@ class TimetableItemDetailViewModel @Inject constructor(
             )
     private val viewBookmarkListRequestStateFlow =
         MutableStateFlow<ViewBookmarkListRequestState>(ViewBookmarkListRequestState.NotRequested)
+    private val currentDescriptionLanguageStateFlow: MutableStateFlow<Lang?> =
```

Comment:
Could you name this variable selectedDescriptionLanguageStateFlow? I think currentDescriptionLanguageStateFlow is a little wrong name because we show our default lang 🙏